pyanaconda/network.py

```diff
@@ -34,6 +34,7 @@
 from uuid import uuid4
 import itertools
 import glob
+import logging
```

Comment:
This is needed in the function `setup_ifcfg_log`. Is the function fine as it is?